web_common/web_common/templates/header.html

```diff
@@ -77,7 +77,15 @@
     </a>
   {% endif %}
 
-  <div id="header-flex"></div>
+  {% if userdata['is_developer'] == 1 %}
+    <div class="benchmark-caret"></div>
+    <a class="header-link" href="{{ benchmark_base_url }}">
+      Benchmark
+    </a>
+  {% endif %}
+
+
+    <div id="header-flex"></div>
```

Comment:
Can you remove two spaces from this line so it lines up in the correct place? Sometimes I find IntelliJ has a hard time with html. Maybe try it in emacs?